### PR TITLE
Main -> Develop - v2.12.0

### PR DESCRIPTION
**Pull request by orakili**
## Features

- RW-954: job tagging test form + improvements to job tagging

## Chores

- OPS-10398: update CSP
- Composer update

### CODE REVIEW

File: config/ocha_ai.settings.yml [hunk 1]
```diff
@@ -38,6 +38,17 @@ plugins:
       max_tokens: 8192
       api_secret: REPLACE_WITH_SECRET
       role_arn: null
+    aws_bedrock_titan_embed_text_v2:
```
Comment (cafuego) [inline config/ocha_ai.settings.yml:41]: ah

File: jobtagger/tagjob.php
Comment (cafuego) [inline jobtagger/tagjob.php:1]: I don't like this not being a drush command.

**Review by cafuego — APPROVED** [review]
Approved, but this will not work on prod until I add AWS settings for the model. Those are also not present on non-prod, so does that imply this isn't tested?

File: config/ocha_ai.settings.yml (hunk 1 above)
Comment (cafuego) [reply config/ocha_ai.settings.yml:41]: Ok, this uses the exiting bedrock settings so should be all good, eh?

File: jobtagger/tagjob.php
Comment (orakili) [reply jobtagger/tagjob.php:1]: This is not going to be run normally. Pinging @attiks to see if that should be removed or refactored in a drush command later.

File: config/ocha_ai.settings.yml (hunk 1 above)
Comment (orakili) [reply config/ocha_ai.settings.yml:41]: Yes, it inherits the AWS settings from the parent plugin so there is nothing new to add.

File: jobtagger/tagjob.php
Comment (attiks) [reply jobtagger/tagjob.php:1]: Was for testing only, I'll remove it

Comment (attiks) [reply jobtagger/tagjob.php:1]: PR at https://github.com/UN-OCHA/rwint9-site/pull/835

**Pull request merged**